encord/user_client.py

```diff
@@ -838,6 +838,23 @@ def get_storage_item(self, item_uuid: UUID, sign_url: bool = False) -> StorageIt
         """
         return StorageItem._get_item(self._api_client, item_uuid, sign_url)
 
+    def get_storage_items(self, item_uuids: List[UUID], sign_url: bool = False) -> List[StorageItem]:
+        """
+        Get storage items by their UUIDs, in bulk. Useful for retrieving multiple items at once, e.g. when getting
+        items pointed to by :attr:`encord.orm.dataset.DataRow.backing_item_uuid` for all data rows of a dataset.
+
+        Args:
+            item_uuids: list of UUIDs of items to retrieve.
+
```

Comment:
Nit: 'sign_url' technically undocumented.
Could be like:
"sign_url: retrieve signed urls for each item"

Reply:
Added the docstring, thanks for being vigilant there!